halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/persistentStorage/S3PersistentStore.java

```diff
@@ -25,6 +25,7 @@
 @Data
 @EqualsAndHashCode(callSuper = false)
 public class S3PersistentStore extends PersistentStore {
+  private String accountName;
```

Comment:
Is this to avoid duplicating the secret key fields here? We originally took this approach with GCS but dropped it because it required granting our keys super extra privileges in clouddriver that weren't required for deploying infrastructure, only managing buckets